fem/pfespace.cpp

```diff
@@ -1278,9 +1278,11 @@ const FiniteElement *ParFiniteElementSpace::GetFaceNbrFaceFE(int i) const
 
 void ParFiniteElementSpace::Lose_Dof_TrueDof_Matrix()
 {
+#if HYPRE_ROWCOLPART_OWNER
```

Comment:
How about we replace this `#if HYPRE_ROWCOLPART_OWNER` statement with
```c++
#ifdef hypre_ParCSRMatrixOwnsRowStarts
```
This should work with all hypre commits without the need of manually setting `HYPRE_ROWCOLPART_OWNER`.

Reply:
I hadn't thought about using the removal of the definitions as the flag. What do you think about having the definition of `HYPRE_ROWCOLPART_OWNER` to be along the lines of
```
#ifndef HYPRE_ROWCOLPART_OWNER
#ifdef hypre_ParCSRMatrixOwnsRowStarts
#define HYPRE_ROWCOLPART_OWNER 1
#else 
#define HYPRE_ROWCOLPART_OWNER 0
#endif
#endif
```
(or even removing the ifndef)
which would allow it to be overridden if needed (although it should be completely atomic, and shouldn't need to be overridden), while making it clear in the code what the flag corresponds with (and requires only a few line change from the current branch)

Reply:
That works too, however, we should not define macros starting with `HYPRE` in MFEM -- we should use `MFEM_HYPRE` instead.